Zoo/include/Dog.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "Location.h"
+#include "Util.h"
+#include "Animal.h"
+
+
+
+
+class Dog :public Animal {
+
+	const Location rightMove = { 3, 0 };
```

Comment:
it's a bad practice to save movements  here, you are saving 10 members for each object